File: whitelist-owner.ts
import { PrismaClient } from '@prisma/client';
const prisma = new PrismaClient();

async function main() {
  await prisma.whitelist.upsert({
    where: { discordId: '903327749534523452' },
    update: {},
    create: {
      discordId: '903327749534523452',
      addedBy: '903327749534523452',
      role: 'owner',
      reason: 'Bot owner',
    },
  });

  await prisma.user.upsert({
    where: { discordId: '903327749534523452' },
    update: { isWhitelisted: true, isBotOwner: true },
    create: {
      discordId: '903327749534523452',
      username: 'owner',
      isWhitelisted: true,
      isBotOwner: true,
    },
  });

  console.log('Owner whitelisted successfully');
  await prisma.$disconnect();
}

main();

File: whitelist_owner.py
from prisma import Prisma
import asyncio


async def whitelist_owner():
    db = Prisma()
    await db.connect()

    owner_id = "903327749534523452"

    await db.whitelist.upsert(
        where={"discordId": owner_id},
        data={
            "create": {
                "discordId": owner_id,
                "addedBy": owner_id,
                "role": "owner",
                "reason": "Bot owner",
            },
            "update": {},
        },
    )

    try:
        await db.user.upsert(
            where={"discordId": owner_id},
            data={
                "create": {
                    "discordId": owner_id,
                    "username": "owner",
                    "isWhitelisted": True,
                    "isBotOwner": True,
                },
                "update": {
                    "isWhitelisted": True,
                    "isBotOwner": True,
                },
            },
        )
    except Exception:
        pass

    print(f"Owner {owner_id} whitelisted successfully")
    await db.disconnect()


if __name__ == "__main__":
    asyncio.run(whitelist_owner())
